src/main/java/cms/gov/madie/terminology/util/TerminologyServiceUtil.java

```diff
@@ -0,0 +1,50 @@
+package cms.gov.madie.terminology.util;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.web.util.UriComponentsBuilder;
+
+import com.nimbusds.oauth2.sdk.util.StringUtils;
```

Comment:
Why oauth2 instead of Apache Commons?

Reply:
@jkotanchik-SB I'll change it, I didn't notice it, you know it was IDE automatic suggestion thing..

Reply:
No problem, was thinking that was the case.